2.0/README.md

```diff
@@ -0,0 +1,500 @@
+* ClosePath // This is a new polygon because area is positive!
+* MoveTo(+2,-7) // NOTE THAT THIS IS RELATIVE TO LAST LINETO!
+* LineTo(+0,+4)
+* LineTo(+4,+0)
+* LineTo(+0,-4) // Cursor at 17,13
+* ClosePath // This is an interior ring because area is negative!
+
+### 4.4. Raster Encoding
+
+Currently the use of `raster` within features is considered **experimental**. Encoders MAY support this field, but are not required to do so at this time. In future releases of this specification a more detailed set of requirements for `raster` support may be defined. It is likely in the future that a format or set of formats for images or rasters will be specified as supported for this field.
+
+### 4.5. Feature Attributes
+
+Feature attributes are encoded as pairs of integers in the `tag` field of a feature. The first integer in each pair represents the index of the key in the `keys` set of the `layer` to which the feature belongs. The second integer in each pair represents the index of the value in the `values` set of the `layer` to which the feature belongs. Every key index MUST be unique within that feature such that no other attribute pair within that feature has the same key index. A feature MUST have an even number of `tag` fields. A feature `tag` field MUST NOT contain a key index or value index greater than the number of elements in the layer's `keys` or `values` set, respectively.
+
+### 4.6. Example
+
+For example, a GeoJSON feature like:
+
+```json
+{
+    "type": "FeatureCollection",
+    "features": [
+        {
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -8247861.1000836585,
+                    4970241.327215323
+                ]
+            },
+            "type": "Feature",
+            "properties": {
+                "hello": "world",
+                "h": "world",
+                "count": 1.23
+            }
+        },
+        {
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -8247861.1000836585,
+                    4970241.327215323
+                ]
+            },
+            "type": "Feature",
+            "properties": {
+                "hello": "again",
+                "count": 2
+            }
+        }
+    ]
+}
+```
+
+Could be structured like:
+
+```js
+layers {
+  version: 2
+  name: "points"
+  features: {
+    id: 1
+    tags: 0
+    tags: 0
+    tags: 1
+    tags: 0
+    tags: 2
+    tags: 1
+    type: Point
+    geometry: 9
+    geometry: 2410
+    geometry: 3080
+  }
+  features {
+    id: 1
+    tags: 0
+    tags: 2
+    tags: 2
+    tags: 3
+    type: Point
+    geometry: 9
+    geometry: 2410
+    geometry: 3080
+  }
+  keys: "hello"
+  keys: "h"
+  keys: "count"
+  values: {
+    string_value: "world"
+  }
+  values: {
+    double_value: 1.23
+  }
+  values: {
+    string_value: "again"
+  }
+  values: {
+    int_value: 2
+  }
+  extent: 4096
+}
+```
```

Comment:
I'd put the keys and values between the name and features, as there's less jumping around where the reader has to look.